src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -439,7 +441,7 @@ internal void SetRequestHeaders()
                 {
                     _requestHeaders = slist;
                     SetCurlOption(CURLoption.CURLOPT_HTTPHEADER, slist);
-                    VerboseTrace("Set headers");
+                    EventSourceTrace();
```

Comment:
There's not really any relevant state to include, but at least this way we trace that the method was used.